website/src/pages/en/subgraphs/developing/creating/subgraph-manifest.mdx

```diff
@@ -532,3 +532,16 @@ To preserve the complete history of entity states:
 indexerHints:
   prune: never
 ```
+## SpecVersion Releases
```

Comment:
Do you have "Format on save" enabled? The format seems wrong here, there should be a space between the code block and this heading.